plugins/plugin-core-event-listener-state-storage/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@syntest/plugin-core-event-listener-state-storage",
+  "version": "0.1.0",
```

Comment:
Shouldn't the first version be 0.0.1 to make the first published version be 0.1.0?

Reply:
Fixed.